libraries/AP_HAL_Linux/Perf.cpp

```diff
@@ -169,6 +169,12 @@ void Perf::end(Util::perf_counter_t pc)
         perf.max = elapsed;
     }
 
+    if (!perf.count) {
```

Comment:
perf.total is unused. On the other hand we use perf.count below for a divider. Any increasing series will eventually wrap, even for uint64_t. However doing the math for a 8k events/sec ( max I can imagine us using with this stuff) it would take in the order of million years to be able to wrap. So yes, I maybe was over cautious when I saw the division :)